services/graphql/src/graphql/resolvers/report.js

```diff
@@ -0,0 +1,33 @@
+const Reporting = require('../../services/reporting');
```

Comment:
This is nit-picky, but this should be lowercased and probably be called `reportingService`. Studly caps are generally used to signify objects that can be constructed... in this case it's already an instance.

Reply:
Also, a general thought... should reporting be an entirely separate service under the `services/reporting` folder?